style.css

```diff
@@ -1,13 +1,72 @@
-h1,.type-zip,.submit{
+h1,.type-zip{
+
 	text-align: center;
 	font-family: 'Denk One', sans-serif;
 }
 
+.type-zip{
+	font-size: 20px;
+}
+
+header{
+	padding:20px;
+	background-color: orange;
+	margin-right: 50px;
+	margin-left:50px;
+	margin-bottom: 20px;
+}
+
+h1{
+	color:white;
+	margin-right: 350px;
+}
+
+.logo{
+	float: left;
+	margin-left:100px;
+
```

Comment:
I try to keep white space as uniform as possible. May want to remove some of these blank lines.